lib/node_modules/@stdlib/math/base/special/bernoullif/src/main.c

```diff
@@ -0,0 +1,87 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/bernoullif.h"
+#include "stdlib/math/base/assert/is_odd.h"
+#include "stdlib/constants/float32/ninf.h"
+#include "stdlib/constants/float32/pinf.h"
```

Comment:
```suggestion
#include <stdint.h>
```

Because you are using `int32_t`